web/sites/default/settings.php

```diff
@@ -37,3 +29,15 @@
 
 $config['openkj.settings']['api_key'] = '12345';
 $config['openkj.settings']['default_venue'] = '1';
+
+// configuration split
+$config['config_split.config_split.dev_split']['status'] = TRUE;
```

Comment:
I think this needs to be wrapped in some sort of conditional logic, such as...

`if ($settings['server_environment'] === 'live') {
  $config['config_split.config_split.development']['status'] = FALSE;
}
else {
  $config['config_split.config_split.development']['status'] = TRUE;
}`